### DISCUSSION TITLE
[SKIA] Recycled TextBox drops binding

### DISCUSSION THREAD

**Discussion by maaxe** [unoplatform/uno#13941] [Q&A]
I guess this is related to [13125](https://github.com/unoplatform/uno/issues/13125)

When I first navigate to a view and edit a TextBox everyting works as expected. But if I navigate to another view and then back to the same view and edit the textbox, the new value is not set. If I then edit the same TextBox again, the new value is set (i.e. need to edit the textbox twice in order for the value to be set in my ViewModel). 

If I disable pooling `FrameworkTemplatePool.IsPoolingEnabled = false` everything works as expected.

Is there som workaround to this that dont include disabling pooling?

Tested on 4.10.13 and 4.11.0-dev.107

**Comment by jeromelaban** [comment]
Thanks for the report. This seems incorrect, could you open an issue [with a repro](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-howto-create-a-repro.md)?

**Reply by maaxe** [reply]
With the latest templates I was not able to recreate the problem. But after changing tagetframework from net7 to net6 I was able to reproduce it. When targeting net7 the usercontrols were not reused for some reason.

[Uno_binding_test.zip](https://github.com/unoplatform/uno/files/12857150/Uno_binding_test.zip)

**Reply by jeromelaban** [reply]
Thanks, make sure to open an issue and test with the latest preview version of Uno to see if that may have already been fixed.

**Reply by maaxe** [reply]
#13942
